### ISSUE TITLE
i18n

### ISSUE THREAD

**Issue by hamoid** [tomitrescak/meteor-tomi-upload-jquery#5]
Has anyone worked on making this package easy to translate? I see strings in HTML and strings in JavaScript...

**Comment by tomitrescak** [comment]
No, currently there is no support for i18n, pull requests are welcome. I would not like to see any package dependencies and custom solution must be deployed, similar to accounts-ui-i18n where a custom language can be easily registered. Thanks.
